src/features/productDetails/ProductDetails.tsx

```diff
@@ -1,13 +1,14 @@
-import {useMemo, useState} from "react";
+import React, {Suspense, useMemo, useState} from "react";
 import {formattedPrice} from "../../utils/productUtils.ts";
 import {Link, useParams} from "react-router-dom";
 import {useQuery} from "@tanstack/react-query";
 import {getProductWithId} from "../../services/api.ts";
 import {useCartStore} from "../cart/cartStore.ts";
-import ProductDetailsLoader from "./ProductDetailsLoader.tsx";
-import AppError from "../error/AppError.tsx";
 import {useWishlistStore} from "../wishlist/wishlistStore.ts";
 
+const AppError = React.lazy(() => import("../error/AppError.tsx"));
+const ProductDetailsLoader = React.lazy(() => import("./ProductDetailsLoader.tsx"));
```

Comment:
I think you can `import("./ProductDetailsLoader.tsx")` `as` something* ,i.e., give it the import an alias then use the alias in this line